webapp/app/pods/logged-in/project/translation/comments/controller.ts

```diff
@@ -46,6 +59,25 @@ export default class CommentsController extends Controller {
     });
   }
 
+  @action
+  async deleteComment(comment: {id: string}) {
+    try {
```

Comment:
I refactored this lately, you can see in the other mutation in the file that we don’t use "error throwing" to control the flow.

I think you can simply assign the response of `this.apolloMutate.mutate` in a variable to check for errors.